tests/tester.js

```diff
@@ -101,54 +99,18 @@ const skipVM = [
   'callstatelessToNameRegistrator0',
   'callstatelessToReturn1',
   'createNameRegistrator',
-  'randomTest643' // TODO fix this
+  'randomTest643', // TODO fix this
 ]
 
-if (argv.r) {
-  randomized(argv.r, argv.v)
-} else if (argv.s) {
+if (argv.state) {
   runTests('GeneralStateTests', argv)
-} else if (argv.v) {
-  runTests('VMTests', argv)
-} else if (argv.b) {
+} else if (argv.blockchain) {
   runTests('BlockchainTests', argv)
+} else if (argv.vm) {
+  runTests('VMTests', argv)
 }
 
-// randomized tests
```

Comment:
I found this randomized test script runner under the flag `-r` that didn't seem to be used anywhere and didn't work for me, so I removed it. If anyone knows of its use, please let me know and I'd be happy to document it, but otherwise I'll assume it's no longer of use.